crates/subspace-networking/src/request_response_handler.rs

```diff
@@ -0,0 +1,232 @@
+// This file is part of Substrate.
```

Comment:
Just like I mentioned in our conversation with every single new contributor to the codebase: copy file from source with 0 changes in one commit (preferrably specifying which commit it was taken from at the source repo) and modify it in a separate commit instead of merging everything into one commit.

Reply:
Added a copied file commit and rebased.